animations/src/main/java/com/chiralbehaviors/CoRE/meta/models/ProductModelImpl.java

```diff
@@ -75,6 +84,57 @@ public void authorize(Aspect<Product> aspect, Attribute... attributes) {
         }
     }
 
+    /* (non-Javadoc)
+     * @see com.chiralbehaviors.CoRE.meta.NetworkedModel#authorize(com.chiralbehaviors.CoRE.ExistentialRuleform, com.chiralbehaviors.CoRE.relationship.Relationship, com.chiralbehaviors.CoRE.agency.Agency)
+     */
+    @Override
+    public void authorize(Product ruleform, Relationship relationship,
+                          Agency authorized) {
+        AgencyProduct a = new AgencyProduct(kernel.getCoreAnimationSoftware());
```

Comment:
Pray tell how on earth A R.Inverse P is any differently stored in a database table than P R.Inverset A.